src/cloud.c

```diff
@@ -281,37 +303,27 @@ int cloud_register_device(const char *id, const char *name)
  */
 int cloud_unregister_device(const char *id)
 {
-	amqp_bytes_t queue_cloud;
 	json_object *jobj_unreg;
 	const char *json_str;
 	int result;
 
-	queue_cloud = mq_declare_new_queue(MQ_QUEUE_FOG_IN);
-	if (!queue_cloud.bytes) {
-		l_error("Error on declare a new queue");
-		return -1;
-	}
-
 	jobj_unreg = parser_unregister_json_create(id);
-	if (!jobj_unreg) {
-		amqp_bytes_free(queue_cloud);
+	if (!jobj_unreg)
 		return KNOT_ERR_CLOUD_FAILURE;
-	}
+
 	json_str = json_object_to_json_string(jobj_unreg);
 
 	headers[0].value.value.bytes = amqp_cstring_bytes(user_auth_token);
 
-	result = mq_publish_persistent_message(queue_cloud,
-					       MQ_EXCHANGE_FOG_IN,
-					       MQ_CMD_DEVICE_UNREGISTER,
-					       headers, 1,
-					       MQ_MSG_EXPIRATION_TIME_MS,
-					       json_str);
+	result = mq_publish_direct_message(MQ_EXCHANGE_DEVICE,
+					   MQ_CMD_DEVICE_UNREGISTER,
+					   headers, 1,
```

Comment:
Add a comment informing whats is 1 or why it should be 1.

Reply:
I put a disclaimer informing about the AMQP settings of the sent message